include/swift/Serialization/ModuleFormat.h

```diff
@@ -643,7 +643,8 @@ namespace input_block {
     IMPORTED_HEADER_CONTENTS,
     MODULE_FLAGS, // [unused]
     SEARCH_PATH,
-    FILE_DEPENDENCY
+    FILE_DEPENDENCY,
+    PARSEABLE_INTERFACE
```

Comment:
Nitpick, here and throughout (sorry for all the back-and-forth): can you please add `path` to all of these? When I originally saw this, I assumed this was going to add the full contents of the interface file to the .swiftmodule 😅

Reply:
I changed the one that I added, we can do the rest in a follow-up commit.